wordpress-mobile/WordPress-iOS
# Remote Feature Flags: Point to the new endpoint

Closes #19308
WPKit PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/539
Ref: pbArwn-57S-p2

## Description
This PR only points to the new WordPressKit version. This new version points to the new feature flag endpoint.

## Testing Instructions

1. Manually modify the app's build number to an integer
2. Install the WordPress app
3. Launch the app
4. Observe the logs looking for a log starting with "Successfully updated local feature flags"
5. Make sure the feature flags retrieved match the WPiOS flags in the MC tool
6. Repeat the same steps for Jetpack

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
